### PR TITLE
container files/axios/boot

### PR DESCRIPTION
**Pull request by vantran23**
I create all the container (pages) files and put them in App.js and added the Navbar component.
I installed Axios and Bootstrap.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,47 @@
...
+
+class App extends React.Component {
+
+  render(){
+    return (
+      <div>Hello App.js Page</div>
+      /*
+      <StateProvider>
+        <HashRouter>
+        <Route path='/' component={Navbar} />
```
Comment (mikeboyle) [inline src/App.js:27]: This is a very small nitpick but I think`<Route />` be indented inside `<HashRouter />`

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,47 @@
...
+import { HashRouter, Route,  Switch} from "react-router-dom";
+
+
+//pages
+
+import Closet from './containers/closet';
+import Opening from './containers/opening';
+import addItem from './containers/addItem';
+import mixNmatch from './containers/mixNmatch';
+import Calendar from './containers/calendar';
```
Comment (mikeboyle) [inline src/App.js:11]: Very tiny nitpick but should these be capitalized like `AddItem` and `MixNMatch` or is there a good reason not to?

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,47 @@
...
+    return (
+      <div>Hello App.js Page</div>
+      /*
+      <StateProvider>
+        <HashRouter>
+        <Route path='/' component={Navbar} />
+          <Switch>
+            <Route path="/" exact strict component={Closet} />
+            <Route path="/opening" exact strict component={Opening} />
+            <Route path="/additem" exact strict component={addItem} />
```
Comment (mikeboyle) [inline src/App.js:31]: Is `/additem` the form for adding a piece of clothing? Will this also still be in a modal?

**Review by mikeboyle — CHANGES_REQUESTED** [review]
Looks good! One question and a couple of small style nitpicks.

**Review by rgreen11 — CHANGES_REQUESTED** [review]
Looks good. Before I merge can you capitalize the names for the files being imported and Wrap the App in a HashRouter

**Pull request closed without merging**